app/views/fact_values/_fact.html.erb

```diff
@@ -8,14 +8,13 @@
         <% end %>
       </td>
   <% end %>
-  <td>
+  <td class="ellipsis">
```

Comment:
true, I haven't found any with such long names even with nested facts from chef but I'll remove it